controllers/compaction/reconciler.go

```diff
@@ -201,13 +206,18 @@ func (r *Reconciler) reconcileJob(ctx context.Context, logger logr.Logger, etcd
 				RequeueAfter: 10 * time.Second,
 			}, fmt.Errorf("error while deleting failed compaction job: %v", err)
 		}
+		druidmetrics.CompactionJobCounterTotal.With(prometheus.Labels{druidmetrics.LabelSucceeded: druidmetrics.ValueSucceededFalse}).Inc()
```

Comment:
Please also record `druidmetrics.CompactionJobDurationSeconds.With(prometheus.Labels{druidmetrics.LabelSucceeded: druidmetrics.ValueSucceededFalse}).Set(<total time elapsed since the job was started>)`. It is important to record the total time taken by failed jobs as well, in addition to total number of jobs that failed.

Since the job status will not record `completionTime` for non-successful jobs (see [documentation](https://pkg.go.dev/k8s.io/api/batch/v1@v0.26.1#JobStatus.CompletionTime)), you will have to calculate the time elapsed since `job.status.startTime` and record that as the metric value here.

Reply:
Done